content/en/docs/components/training/installation.md

```diff
@@ -1,99 +1,41 @@
 +++
 title = "Installation"
-description = "How to install the Training Operator"
+description = "How to install the Kubeflow SDK"
 weight = 20
 +++
 
-This guide describes how to install the Training Operator on your Kubernetes cluster.
-The Training Operator is a lightweight Kubernetes controller that orchestrates the
-appropriate Kubernetes workloads to perform distributed ML training and fine-tuning.
+This guide describes how to install the Kubeflow Python SDK to manage your TrainJobs.
 
 ## Prerequisites
 
-These are the minimal requirements to install the Training Operator:
+Ensure that you have access to a Kubernetes clusters with the Kubeflow Training
+control plane installed. If it is not set up yet, follow
+[the installation guide](/docs/components/training/admin-guides/installation) to quickly deploy
+Kubeflow Training on a local Kind cluster.
```

Comment:
```suggestion
[the installation guide](/docs/components/training/admin-guides/installation) to deploy
Kubeflow Training.
```
No reason it needs to be kind, and if there are webhooks it won't be that quick :P

Reply:
I intentionally add this message to tell that it is super easy to deploy Kubernetes locally, and quickly try Kubeflow Training. I don't want to scare our ML users with "Kubernetes" dependency.
@kubeflow/wg-training-leads @vsoch @astefanutti @franciscojavierarceo @StefanoFioravanzo Any thoughts on how we can phrase it better in docs ?

Reply:
I agree with @vsoch suggestion, it doesn't seem there is a need to be too specific here, "quickly deploy Kubeflow Trainer" straight makes it even less scary here and do not exclude non-Kind options.

Reply:
@vsoch @astefanutti ?